frontend/src/FindQuiz.js

```diff
@@ -0,0 +1,185 @@
+        });
+    }
+
+    render() {
+        return (
+            <div id="Quiz">
+                <h1>Find restaurants near you</h1>
+                <form onSubmit={this.handleSubmit}>
+                    <label id="q1">
+                        1. What are you in the mood for?
+                        {/* TODO Allow multiple boxes to be checked */}
+                        <select value={this.state.q1.value} required onChange={this.handleChange} name="q1">
+                            <option value="Bar & Grill">Bar & Grill</option>
+                            <option value="Sandwiches">Sandwiches</option>
+                            <option value="Pizza">Pizza</option>
+                            <option value="Fast Food">Fast Food</option>
+                            <option value="Breakfast">Breakfast</option>
+                            <option value="Steakhouse">Steakhouse</option>
+                            <option value="Fine Dining">Fine Dining</option>
+                            <option value="Sushi">Sushi</option>
+                            <option value="Seafood">Seafood</option>
+                            <option value="Barbeque">Barbeque</option>
+                            <option value="American">American</option>
+                            <option value="Mexican">Mexican</option>
+                            <option value="South American">Mexican</option>
+                            <option value="Italian">Italian</option>
+                            <option value="Eastern European">Eastern European</option>
+                            <option value="Mediterranean">Mediterranean</option>
+                            <option value="Middle Eastern">Middle Eastern</option>
+                            <option value="Indian">Indian</option>
+                            <option value="Chinese">Chinese</option>
+                            <option value="Japanese">Japanese</option>
+                            <option value="Korean">Korean</option>
+                            <option value="Southeast Asian">Southeast Asian</option>
+                        </select>
+                    </label>
+                    <br />
+                    <label id="q2">
+                        2. How much are you willing to spend?&emsp;&emsp;
+                        <select value={this.state.q2.value} required onChange={this.handleChange} name="q2">
+                            <option value="$">$</option>
+                            <option value="$$">$$</option>
+                            <option value="$$$">$$$</option>
+                            <option value="$$$$">$$$$</option>
+                        </select>
+                    </label>
+                    <br />
+                    <label id="q3">
+                        3. Maxmimum Distance (mi.)&emsp;
+                        <input type="number" defaultValue="1" min="1" value={this.state.q2.value} name="q3"
```

Comment:
We may want to set this to be a maximum of 24 or 25 miles. Yelp API won't return results if the distance is over 40km (24.8 miles), and reducing the radius to 40km in the backend may be confusing to users (e.g. looking for a restaurant 50 miles away and nothing shows up). Quote from [Yelp API Documentation](https://www.yelp.com/developers/documentation/v3/business_search):

"""Optional. A suggested search radius in meters. This field is used as a suggestion to the search. The actual search radius may be lower than the suggested radius in dense urban areas, and higher in regions of less business density. If the specified value is too large, a AREA_TOO_LARGE error may be returned. The max value is 40000 meters (about 25 miles)."""

Reply:
Good call. I'll fix that in the next commit.